pilot/pkg/model/service.go

```diff
@@ -84,6 +86,13 @@ type Service struct {
 	// or use the passthrough model (i.e. proxy will forward the traffic to the network endpoint requested
 	// by the caller)
 	Resolution Resolution
+
+	// ID represents the location of the config used to create this Service. Optional.
+	// For k8s, it is namespace:name
+	ID string `json:"name,omitempty"`
+
+	// Time records the resource time (modified time if available, creation otherwise), if available.
+	Time time.Time `json:"mtime,omitempty"`
```

Comment:
Let's be consistent with the json name.  Either call them both `modifiedTime` or just `time`.

Reply:
CreationTime is what we need for conflict resolution

Reply:
+1 to `CreationTime`